sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedManualValidationInfo.java

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormWord;
+import com.azure.ai.formrecognizer.models.OperationResult;
+import com.azure.ai.formrecognizer.models.RecognizedForm;
+import com.azure.ai.formrecognizer.models.TextContentType;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.polling.PollerFlux;
+import reactor.core.publisher.Mono;
+
+/*
+ * This sample demonstrates how to output the information that will help with manually
+ * validating your output from recognize custom forms.
+ */
+public class AdvancedManualValidationInfo {
+
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        FormRecognizerAsyncClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential("{api_key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildAsyncClient();
+
+        String modelId = "{model_Id}";
+        String filePath = "{analyze_file_path}";
+        PollerFlux<OperationResult, IterableStream<RecognizedForm>> trainingPoller =
```

Comment:
I think we determined we were going to use recognize custom forms from file for the manual validation sample